pkgs/tools/misc/loop/default.nix

```
@@ -1,16 +1,16 @@
 { stdenv, fetchFromGitHub, rustPlatform }:
 
 rustPlatform.buildRustPackage {
-  name = "loop-unstable-2018-12-04";
+  name = "loop-unstable-2020-07-08";
```

Since this is being updated, it's also a good opportunity to modernize this:

```suggestion
  pname = "loop";
  version = "unstable-2020-07-08";
```